state_machine/steps

```diff
@@ -0,0 +1,164 @@
+#activate workspace
```

Comment:
is this for you guys or should it be in the repo?

Reply:
I think this was supposed to be a bash file initially, but it doesnt run like a bash file because it has too much uncommented task, I think it's just a list of conditional commands that need to be done in order while manually doing a couple of things. I would leave this as is